api/middleware/cf_user_test.go

```diff
@@ -126,7 +126,7 @@ var _ = Describe("CfUserMiddleware", func() {
 		})
 
 		It("delegates to the next middleware", func() {
```

Comment:
We should rephrase these test descriptions as the new purpose of the middleware is to block requests instead of just adding a warning and then delegating to the next middleware.